packages/mmh3/meta.yaml

```diff
@@ -0,0 +1,29 @@
+package:
+  name: mmh3
+  version: 4.1.0
+source:
+  url: https://files.pythonhosted.org/packages/63/96/aa247e82878b123468f0079ce2ac77e948315bab91ce45d2934a62e0af95/mmh3-4.1.0.tar.gz
+  sha256: a1cf25348b9acd229dda464a094d6170f47d2850a1fcb762a3b6172d2ce6ca4a
+about:
+  PyPI: https://pypi.org/project/mmh3
+  summary:
+    Python extension for MurmurHash (MurmurHash3), a set of fast and robust
+    hash functions.
+  license:
+    'MIT License  Copyright (c) 2011-2023 Hajime Senuma  Permission is hereby
+    granted, free of charge, to any person obtaining a copy of this software and associated
+    documentation files (the "Software"), to deal in the Software without restriction,
+    including without limitation the rights to use, copy, modify, merge, publish,
+    distribute, sublicense, and/or sell copies of the Software, and to permit persons
+    to whom the Software is furnished to do so, subject to the following conditions:  The
+    above copyright notice and this permission notice shall be included in all copies
+    or substantial portions of the Software.  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT
+    WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES
+    OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO
+    EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES
+    OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+    FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+    THE SOFTWARE.'
```

Comment:
If this is the normal text of the MIT license, could you shorten it to just `license: MIT`?

Reply:
lol of course :) this is what came down with the skeleton command and I think all it adds is the author's name, but MIT should be just fine.